distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -137,10 +138,12 @@ Text searches are by default case insensitive, but case sensitive searches are a
 +
 *Between*: set a beginning and end time to search between.
 
-[[_spatial_search_details]]Spatial Search Details:: Search by latitude/longitude or the USNG using a line, polygon, point-radius, or bounding box.
+[[_spatial_search_details]]Spatial Search Details:: Search by latitude/longitude, USNG, or UTM using a line, polygon, point-radius, or bounding box.
 
 [[_type_search_details]]Type Search Details:: Search for specific content types.
 
+[[_sort_search_details]]Sort Parameters:: Specify which Attribute and order to search sort the Query results.
```

Comment:
Maybe a little better wording:
+[[_sort_search_details]]Sort Parameters:: Specify which Attribute(s) and it's sort order for Query result sorting.